hudi-common/src/test/java/org/apache/hudi/common/testutils/HoodieTestTable.java

```diff
@@ -289,6 +289,14 @@ public HoodieTestTable moveInflightCommitToComplete(String instantTime, HoodieCo
     return this;
   }
 
+  public void moveCompleteCommitToInflight(String instantTime) throws IOException {
```

Comment:
Is this method also used for MOR testing with compaction? In that case, deleteCommit should be called instead of deleteDeltaCommit right.

Reply:
yeah, maybe we can extend it when there is a necessity.